.github/workflows/wpt-test.yml

```diff
@@ -0,0 +1,83 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "**/*.md"
+
+jobs:
+  test:
+    name: WebPageTest Test Cases
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: |
+          npm install webpagetest
+          npm install jest
+
+      - name: Run WebPageTest with unit tests
+        run: |
+          echo "::group::Unit tests"
+            npm test
+          echo "::endgroup::"
+        env:
+          WPT_SERVER: "webpagetest.httparchive.org"
+          WPT_API_KEY: ${{ secrets.HA_API_KEY }}
```

Comment:
@pmeenan with new server we get an error:

console.error
      WPTAPIError {
        name: 'WPTAPIError',
        code: 301,
        message: 'Moved Permanently'
      }
Please adjust the configuration here.

Reply:
Sorry, looks like the client prefers http over https and the server was set to redirect to https.  I've updated the server to allow connections over http as well.

Reply:
@pmeenan Server responds with 400 now:
```
"statusCode": 400,
"statusText": "An error occurred processing your request (missing API key). If you do not have an API key you can purchase one here: https://product.webpagetest.org/api",
```
Please check the secret.

To test it:
```bash
export WPT_SERVER=webpagetest.httparchive.org
export WPT_API_KEY=INSERT_API_KEY_HERE
npm test
```

Reply:
Sorry about the delay. Not sure why I'm not getting notifications but I'll try to check regularly until it lands.

Not 100% sure why, but it looks like if I pass the key explicitly as part of the options then it works. I submitted the change (hopefully that's ok).

Reply:
@pmeenan Thanks for a hint. Looks OK indeed.